downstream/README.md

```diff
@@ -0,0 +1,238 @@
+
+#### What this should done?
+
+1. Find the source for the new release and upload it to the lookaside cache if it is not already there.
+2. Update the version in specfile
+3. Create the new changelog entry. There are multiple options for this:
+
+- Changelog contains some generic text like `New upstream version X.Y.Z`
+- Changelog is generated from the git history.
+- Changelog is copied from the upstream (GitHub) release.
+- Changelog is generated using a custom command. (Some environment variables can be provided what can be used when the syntax of the first option can't be used.)
+
+### Implementation
+
+#### How we can get the event?
+
+If user install Packit GitHub application, we can get the event via webhook from the GitHub.
+If user does not have Packit installed in upstream, we can react on the fedora-messaging messages created by Upstream Release Monitoring. (Package can be added on https://release-monitoring.org/ .)
+
+#### How to implement it?
+
+We already have this implemented.
+
+What can be added is providing a feedback during the process:
+
+- providing a dashboard view for the job
+- live logs or letting the user know where the job was triggered
+  - When reacting to the comment, we can put a comment with the details (link to the PRs).
+  - When reacting to the release/commit, we can use commit status (check runs) to show some details.
+- When things go bad, there should be some feedback to the user.
+  - Also warnings should be accessible.
+  - This is also true when retriggering the job when the dist-git PRs have already been created. In such scenario, the current service silently ends. (One needs to close those PRs to regenerate them.)
+
+#### How to communicate with the user?
+
+Currently, we create a PR and that's all. We should inform the user about the process (link needed) and there is no information about this job on dashboard.
+
+## Koji builds
+
+### User perspective
+
+When there is a new commit in dist-git, I want a new build for it to be triggered. Optionally, I can set that I want to trigger scratch build instead of the production one or set other arguments when triggering the build (e.g. side-tag).
+
+#### How to enable?
+
+Options:
+
+- I can send a pull-request to a repository containing the list of repositories to build. (The same thing zuul does.)
+- I can add a packit config to the distgit repository and configure the job (can be combined with the previous version so we can filter the projects before checking config presence).
+- A service authenticated by FAS where I can enable the build and, ideally, configure the process.
+
+#### How to trigger?
+
+The build is triggered once there is a new commit in distgit.
+
+Optionally, I can retrigger the action by a commit comment on dist-git.
+
+#### How to enhance the workflow?
+
+In usual case, no enhancement is needed. If we need one, we can use packit config to provide this extra info.
+
```

Comment:
we could post a comment with a link to the build to a PR if HEAD comes from a PR